### ISSUE TITLE
Compact Machines villager trading.

### ISSUE THREAD

**Issue by Crumbology** [Beyond-Reality/Beyond-Reality-Divergence#256]
http://imgur.com/HtEhoSC

This trade is super cheap, a quantum entangler takes a nether star and is purchasable for 8 emeralds.  This is using the Amadron tablet from pneumaticraft, but it does seem like this villager will spawn acc to the compact machines config.

Enables Villager spawns trading PSDs, QEs and World Resizing Cubes. [default: true]
    B:enableVillager=true

**Comment by Dominance** [comment]
My fault for not knowing that Pneumaticraft added such a item. Seems like the easier route would be to just disable the item?

**Comment by Crumbology** [comment]
It seems kinda necessary to advance in pneumaticraft, it's how I got my pcb blueprint and the templates for the assembly line.   It's also a pretty fun mechanic allowing to trade oil/diesel and other PC refinery byproducts for emeralds, and it delivers, a pretty unique system which doesn't seem overpowered in anyway.  It even allows for player trading over long distances.

Anyway I found the PC configs for that and it's all controllable luckily.

config/PneumaticCraft/AmadronOffersPeriodic.cfg
